app/javascript/vue/navigation/top_menu.vue

```diff
@@ -6,7 +6,7 @@
         <img class="logo large" :src="logo.large_url">
       </a>
     </div>
-    <div v-if="teams" class="sci--navigation--top-menu-teams">
+    <div v-if="current_team" class="sci--navigation--top-menu-teams">
```

Comment:
I'm pretty sure `current_team` was renamed to `currentTeam` at some point, so we need to change it to that.